components/support/nimbus-fml/src/workflows.rs

```diff
@@ -703,6 +737,48 @@ mod test {
         Ok(())
     }
 
+    #[test]
+    fn test_validate_command() -> Result<()> {
+        for path in MANIFEST_PATHS {
+            if path.ends_with(".yaml") {
+                println!("{}", path);
+                let manifest = join(pkg_dir(), path);
+                let cmd = ValidateCmd {
+                    loader: Default::default(),
+                    manifest,
+                };
+                validate(&cmd)?;
+            }
+        }
+        Ok(())
+    }
+
+    #[test]
+    fn test_validate_command_fails_on_bad_default_value_for_one_channel() -> Result<()> {
+        let path = "fixtures/fe/invalid/invalid_default_value_for_one_channel.fml.yaml";
+        let manifest = join(pkg_dir(), path);
+        let cmd = ValidateCmd {
+            loader: Default::default(),
+            manifest,
+        };
+        let result = validate(&cmd);
+
+        assert!(result.is_err());
+
+        match result.err().unwrap() {
+            ValidationError(path, error) => {
+                assert_eq!(path, "features/example-feature.enabled".to_string());
```

Comment:
You don't need to do `.to_string()` here. You can compare a `String` and a `&str` for equality.

Reply:
Huh rust clippy usually catches stuff like that and extra `clone`s, I'm surprised it didn't.